### DISCUSSION TITLE
Setting won't save

### DISCUSSION THREAD

**Discussion by RDJay56** [j7126/OctoPrint-Dashboard#363] [Q&A]
I go to Octoprint Settings/Dashboard.  I add the Webcam to the Enabled Wedgets.  

The "Save" button has the circle next to Save moving, I click save and it doesn't stop.  Next time I open Octoprint the Webcam is disabled.

Appreciate any help.

**Comment by j7126** [comment]
Please try the following:
Can you save other settings?
If dashboard is disabled can you save other settings?
If all plugins except dashboard are disabled can you save settings?

**Comment by RDJay56** [comment]
Other setting also wont save.

I disabled all plugins, reenabled Dashboard and it seems fine, "Save" now works.  I'm using Octoprint on an Android so maybe an issue there or I had too many plugins installed and there was a conflict.

Sorry to bother here, and thank you for the suggestions.

Rj

**Reply by j7126** [reply]
It is likely that one of the other plugins was the cause or it was a conflict, if you enable your other plugins one by one you should be able to figure out which one it is.

**Reply by RDJay56** [reply]
Thanks, will do that.
